pkg/processor/sumologicschemaprocessor/processor_test.go

```diff
@@ -1298,3 +1353,15 @@ func newAggregateAttributesConfig(aggregations []aggregationPair) *Config {
 	config.AggregateAttributes = aggregations
 	return config
 }
+
+func newLogFieldsConversionConfig(logFieldsConversion bool) *Config {
```

Comment:
The parameter `logFieldsConversion` does not seem to be used?

Reply:
I updated PR according to our today discussion as well as fixed some bugs. Current configuration looks as suggested by @sumo-drosiek, however I have not change main key `field_attributes`
```
field_attributes:
  trace_id:
    enabled: true
    name: traceid
  span_id:
    enabled: true
    name: spanid
```